src/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "../configs/tsconfig.composite.json",
     "compilerOptions": {
-        "declarationDir": "../typings"
+        "declarationDir": "../typings",
+        "checkJs": false
```

Comment:
Can we adjust this flag after we agree on that as a team? If we decide to drop `checkJs`, that would be best done in `tsconfig.base.json`, and not individual project tsconfigs :slightly_smiling_face:

Reply:
Sorry about that - this was committed accidentally. I turned this flag off as all these (false) errors were really annoying and forgot about it. Lets discuss tomorrow. I'm reverting the change now.

Reply:
Sure, no problemo :slightly_smiling_face: